src/SignalR/docs/specs/TransportProtocols.md

```diff
@@ -18,7 +18,11 @@ Throughout this document, the term `[endpoint-base]` is used to refer to the rou
 
 ## `POST [endpoint-base]/negotiate` request
 
-The `POST [endpoint-base]/negotiate` request is used to establish a connection between the client and the server. There is an optional `negotiateVersion` query string parameter that can be added to the negotiate post request. This determines the protocol version between the server and the client. If omitted, the version is defaulted to zero. Servers can configure their minimum supported protocol version which will determine how they respond to old clients. If configured to be greater than the clients requested version, the server will send an error response indicating that the requested version is not supported by the server. If the server supports the requested protocol version it will set the "negotiateVersion" on the response to the clients version. If a server is configured to accept clients using protocol version 0, then the `connectionToken` is omitted and only the `connectionId` is used internally. The content type of the response is `application/json`. The response to the `POST [endpoint-base]/negotiate` request contains one of three types of responses:
+The `POST [endpoint-base]/negotiate` request is used to establish a connection between the client and the server.
+
+[3.1] There is an optional `negotiateVersion` query string parameter that can be added to the negotiate POST request. This is used to determine the protocol version between the server and the client. If omitted, the server treats the version as zero. Servers can configure their minimum supported protocol version which will determine how they respond to old clients. If configured to be greater than the clients requested version, the server will send an error response indicating that the requested version is not supported by the server. The server will set the "negotiateVersion" on the response to the smaller of either the servers maximum supported version or the clients requested version. The client is free to close the connection if the returned version is not supported by it. If the negotiated version is 0, then the `connectionToken` is omitted from the response and only the `connectionId` is used.
```

Comment:
Can we do some better layout here? I'm thinking something like (just an outline before, we should write prose for it):

Above this, as a top-level heading "Protocol Version", have a section that talks about the concept of protocol versioning. Including the fact that omitting the version negotiation indicates version `0`.

Then, in the negotiate request section, we can use version 0 and version 1 as pivots and shows the two options.